packages/db/prisma/extensions/base.test.ts

```diff
@@ -0,0 +1,69 @@
+import { describe, expect, it } from "vitest";
+
+import { updateAddressData } from "./base";
+
+describe("updateAddressData", () => {
```

Comment:
Same as before: Let's avoid unit testing this internal function as well.

We can check this logic indirectly when testing for the extension functions that use this underneath